zipkin-collector/scribe/pom.xml

```diff
@@ -41,7 +41,7 @@
     </dependency>
 
     <dependency>
-      <groupId>${armeria.groupId}</groupId>
+      <groupId>com.linecorp.armeria</groupId>
```

Comment:
I removed the overriding of the group ID as jitpack has been out of commission a while for things we compile